hazelcast-hibernate5/src/main/resources/phone.home.properties

```diff
@@ -0,0 +1,2 @@
+project.name = ${project.artifactId}
```

Comment:
Shouldn't these be read from `SomeClass.class.getPackage.get*`?

Reply:
I suppose it's not resolved before packaging. For instance, `Package#getImplementationVersion()` will be null if you attempt to get the value for one of the classes in the module in a test.